packages/cashier/src/pages/payment-agent/payment-agent-signup/cancel-wizard-dialog.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Dialog } from '@deriv/components';
+import { localize } from '@deriv/translations';
+
+type TCancelWizardDialogProps = {
+    onConfirm: () => void;
+    onCancel: () => void;
```

Comment:
```suggestion
    onConfirm: VoidFunction;
    onCancel: VoidFunction;
```
@hamid-deriv There is no difference 😅 You can ignore this 🙇🏻